src/main/java/ca/corefacility/bioinformatics/irida/ria/web/analysis/dto/AnalysisDetails.java

```diff
@@ -0,0 +1,131 @@
+package ca.corefacility.bioinformatics.irida.ria.web.analysis.dto;
+
+import java.util.Date;
+
+import ca.corefacility.bioinformatics.irida.model.workflow.submission.AnalysisSubmission;
+
+/**
+ * Used as a response for encapsulating analysis details
+ */
+
+public class AnalysisDetails {
+
+	private String responseCode;
+	private String workflowName;
+	private String version;
+	private String priority;
+	private Long duration;
+	private Date createdDate;
+	private AnalysisSubmission.Priority[] priorities;
+	private boolean emailPipelineResult;
+	private boolean canShareToSamples;
+	private boolean updatePermission;
+	private boolean updateSamples;
+
+	public AnalysisDetails() {
+	}
+
+	public AnalysisDetails(String responseCode, String workflowName, String version, String priority, Long duration,
+			Date createdDate, AnalysisSubmission.Priority[] priorities, boolean emailPipelineResult,
+			boolean canShareToSamples, boolean updatePermission, boolean updateSamples) {
+		this.responseCode = responseCode;
+		this.workflowName = workflowName;
+		this.version = version;
+		this.priority = priority;
+		this.duration = duration;
+		this.createdDate = createdDate;
+		this.priorities = priorities;
+		this.emailPipelineResult = emailPipelineResult;
+		this.canShareToSamples = canShareToSamples;
+		this.updatePermission = updatePermission;
+		this.updateSamples = updateSamples;
+	}
+
+	public String getResponseCode() {
+		return responseCode;
+	}
+
+	public void setResponseCode(String responseCode) {
```

Comment:
Why are you setting a response code here?  If this gets returned it should have a success code.